Update README.pt-br.md

viniciusifsp opened the pull request:
Proposal to update the translation to pt-BR:

- Removed extra spaces
- Some parts of the text probably with more accurate translation
- Added some parts that are suppressed from the original English version

github-education reviewed (changes requested):
**Hi viniciusifsp,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *Uh Oh! I've found some issues with where you have created your files!* 
	*The required markdown file does not exist, please ensure the file `_data/viniciusifsp/viniciusifsp.md` exists*
*Please ensure all changes are contained within the `_data/viniciusifsp/` directory. Invalid file paths:* 

	* `translations/README.pt-br.md`


Feel free to re-request a review from me and I'll come back and take a look!

viniciusifsp commented:
Proposal to update the translation to pt-BR:

- Removed extra spaces
- Some parts of the text probably with more accurate translation
- Added some parts that are suppressed from the original English version

juanpflores commented:
Hi there! thanks for submitting this pull request. We will need you to add yourself in the translations table, in the Portuguese row, to continue and accept this PR
